x-pack/plugins/security_solution/public/detections/components/user_info/index.tsx

```diff
@@ -275,7 +274,7 @@ export const useUserInfo = (): State => {
   }, [dispatch, loading, hasEncryptionKey, isApiEncryptionKey]);
 
   useEffect(() => {
-    if (!loading && canUserCRUD !== capabilitiesCanUserCRUD && capabilitiesCanUserCRUD != null) {
+    if (!loading && canUserCRUD !== capabilitiesCanUserCRUD) {
```

Comment:
I kept the logic intact, just reduced it a little bit. `capabilitiesCanUserCRUD != null` was always returning `true` because `capabilitiesCanUserCRUD` is a boolean.

Reply:
Oh, I see, so I overlooked the type. 👍